internal/proxy/proxy.go

```diff
@@ -582,6 +584,26 @@ func (p *Proxy) createRequestHash(url string, headers string) uint32 {
 	return h.Sum32()
 }
 
+func (p *Proxy) getUserWithRetry(repositoryCloneURL string, logEntry *log.Entry, retry int) (clients.Namespace, error) {
+	var namespace clients.Namespace
+	var error error
```

Comment:
@chmouel this one :) - `var error error` to `var err error` or even better not declare it at all since it is used only inside the for loop

Reply:
Yes in fact this var declaration can be removed with current version. Let me refactor a bit.